flytekit/core/promise.py

```diff
@@ -36,6 +37,33 @@
 from flytekit.models.types import SimpleType
 
 
+def _is_none_valid_instance_of(t: Type) -> bool:
```

Comment:
can you copy these examples into a unit test also?

Reply:
This function is deleted. I use `UnionTransformer.is_optional_type` instead as mentioned above.